Documentation for the Fortanix DSM KMS. (#1713)
kroxylicious/kroxylicious: docs/modules/record-encryption/fortanix-dsm/con-fortanix-dsm-key-creation.adoc

// file included in the following:
//
// assembly-fortanix-dsm.adoc

[id='con-fortanix-dsm-key-creation-{context}']
= Creating Fortanix DSM keys

> As the administrator, create an AES-256 symmetric keys following your key naming convention and a member of the

Reviewer: ```suggestion
As the administrator, create AES-256 symmetric keys following your key naming convention and belonging to the
```
